tests/server_it_test.rs

```diff
@@ -0,0 +1,64 @@
+mod integration {
+    use std::{io::Write, process::*};
+    use fs_extra::dir;
+
+    #[test]
+    fn it_should_process_catalogue_with_correct_output() {
+        // setup
+        cleanup();
+        setup();
+
+        // given
+        let mut process = spawn_pastelogue_server();
+
+        // when
+        let start_processing_json = r#"{"action": "START_PROCESSING", "args": { "path": "./resources/it_test" } }"#;
```

Comment:
już w git ignore widziałem, nie lepiej wrzucić ./resources/it_test skoro są w ignore, do jakieś katalogi ./build/test? wydaje mi się to wtedy czystsze rozwiązanie niż dorzucanie kolejnych katalogów ignorowanych pomiędzy innymi pliczkami ;)